GitUI/BranchTreePanel/RepoObjectsTree.cs

```diff
@@ -526,9 +526,14 @@ private void TxtBranchCriterion_KeyDown(object sender, KeyEventArgs e)
 
         private void OnPreviewKeyDown(object sender, PreviewKeyDownEventArgs e)
         {
-            if (e.KeyCode == Keys.F3)
+            switch (e.KeyCode)
             {
-                OnBtnSearchClicked(null, null);
+                case Keys.F2:
+                    Node.OnNode<Node>(treeMain.SelectedNode, node => node.OnRename());
+                    break;
+                case Keys.F3:
+                    OnBtnSearchClicked(null, null);
```

Comment:
We should call `DoSearch();` instead.
We shouldn't be calling `OnBtnSearchClicked(null, null);` like we are now.